package.json

```diff
@@ -38,10 +38,15 @@
     "generate-docs": "api-documenter markdown --input-folder temp --output-folder docs && rm -rf temp",
     "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite",
     "prepublishOnly": "npm run build",
-    "test": "jest --coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
+    "build-storybook": "build-storybook",
+    "test": "jest --projects tests/jest.unit.config.js --coverage",
+    "coverage:visual": "jest --projects tests/jest.visual.config.js --coverage",
+    "coverage:clean": "rm -rf .nyc_output && rm -rf coverage",
+    "coverage:merge": "istanbul-merge --out coverage/merged/coverage-final.json ./coverage/unit/coverage-final.json  ./coverage/visual-regression/coverage-final.json",
+    "coverage:merge-report": "nyc report --reporter=lcov --reporter=text --temp-dir=./coverage/merged --report-dir=./coverage/merged",
+    "coverage": "npm run coverage:clean && npm run test && npm run coverage:visual && npm run coverage:merge && npm run coverage:merge-report"
```

Comment:
Having all these new `coverage` scripts is a bit confusing. The names `merge` and `merge-report` are pretty similar, I think we should do more to distinguish the two. Is `clean` scoped to just the visual regression coverage? If so, its name should indicate that.

Reply:
merge command is to combine the visual and jest coverage lcov files and the merge-report command is to display them. I was wondering if I should combine them or rename them to something else? The clean command removes the previous coverage folder which includes the visual, jest, and merge coverage files.

Reply:
can we have a bash script instead? I don't think anybody is ever going to call something like `coverage:clean` on its own

Reply:
I agree! We can probably move some stuff to bash script, and in package.json, have three general commands: `test:unit`, `test:visual`, `test:all` or something similar